lib/team_city_formatter/logger.rb

```diff
@@ -13,7 +13,7 @@ def test_suite_finished(test_suite_name)
     end
 
     def test_started(test_name)
-      render_output("##teamcity[testStarted name='#{teamcity_escape(test_name)}' timestamp='#{timestamp}' captureStandardOutput='true']")
+      render_output("##teamcity[testStarted name='#{teamcity_escape(test_name)}' captureStandardOutput='true' timestamp='#{timestamp}']")
```

Comment:
@gricsi is there a specific reason you moved the `captureStandardOutput` attribute here and in `#test_failed`?